config/config.yml

```diff
@@ -1,5 +1,6 @@
 ---
-debug: # true for debugging
+debug: debug # fatal, error, warn, info, debug; defaults to 'fatal'
+logfile: stderr # stderr, stdout, or path; defaults to log/nailed.log
```

Comment:
No, both entries are part of the PR.

Previously, you could only log to a file. Now you can log to stdout or stderr.

'debug' was a mess for itself. It was 'debug: true' in the config.yml, code tested for 'Debug' key, and string values (log levels).